book/src/theme-gallery/index.md

```diff
@@ -114,3 +115,74 @@ It features a dominant partition of space into a bright and a dark side.
 ```
 ![bipartite theme screenshot](./bipartite.png)
 
+
+## Bristol
+_[go to top](#theme-gallery)_
+
+This is a variation of the default theme, to feature academic branding. It is inspired by an old version of David Barton's
+[Beamer theme](https://github.com/dawbarton/UoB-beamer-theme).
+It features a University of Bristol branding by default, however the logos and colour choices can easily be swapped, to tailor the theme to any institution of your choice.
+```typ
+#import "themes/bipartite.typ": *
+#show: slides(
+  // ...
+  theme: bristol-theme(),
+)
+```
+
+### Options
+- `color`: the colour to use for decorative elements, default University of Bristol red
+- `watermark`: file path for a watermark image to span the title slide, default "bristol/watermark.svg"
+- `logo`: file path for a logo image to appear on every slide, default "bristol/logo.svg"
```

Comment:
Maybe note that those default files are shipped with the theme so users don't worry about providing them themselves.